lib/utils/stream.js

```diff
@@ -113,4 +116,8 @@ Stream.prototype.flush = function() {
   this.trigger('done');
 };
 
+Stream.prototype.error = function(error) {
```

Comment:
Yes, they can. They do so via `stream.trigger(errorEvent)`, not through a custom method.